src/App.js

```diff
@@ -14,13 +41,15 @@ const App = () => {
     audioManagerInstance.pause();
   };
 
-  const handleSelect = () => {
-    audioManagerInstance.select(hardcodedRadioUrl1);
+  const handleSelect = (selectedStation) => {
+    console.log(selectedStation);
+    audioManagerInstance.select(selectedStation);
   };
 
   return (
     <div className="App">
       <header className="App-header">Hello!</header>
+      <StationsList stations={radioStationsArray} handleSelect={handleSelect} />
```

Comment:
You might want to use state to keep & manage stations list inside of the component:
https://reactjs.org/docs/hooks-state.html

Reply:
I corrected it :)